test-suite-http-server-tck-gcp-function-http-test/src/test/java/io/micronaut/http/server/tck/gcp/function/GcpFunctionHttpTestServerUnderTest.java

```diff
@@ -54,6 +54,7 @@ public ApplicationContext getApplicationContext() {
 
     @Override
     public void close() throws IOException {
+        server.getApplicationContext().close();
```

Comment:
Instead of this, should we change `InvokerHttpServer` to override close as so:

```java

    @Override
    public void close() {
        this.applicationContext.close();
        stop();
    }
```

Reply:
[I changed it](https://github.com/micronaut-projects/micronaut-gcp/pull/914/commits/c69357d0a88dbc5aad9a7dcdb84ebbb8191c7460). Not sure if it is better.